routes/products.js

```diff
@@ -1,19 +1,23 @@
 const express = require('express');
 const router = express.Router();
-const products = require('../src/data/products');
-const categories = require('../src/data/categories');
+const dataService = require('../services/dataService');
 
+/*
+ * This method returns the complete products array
+ */
 router.get('/all', function(req, res, next) {
-    const response = [];
-    products.products.forEach( product => {
-        categories.categories.forEach( category => {
-            if (product.categoryId === category.id) {
-                product.category = category;
-            }
-        });
-        response.push(product);
-    });
-    res.send(response);
+  const products = dataService.getCombinedProductMap();
+  res.send(Object.values(products));
```

Comment:
Can't we just send the products? So it would be an object or key/paris

Reply:
Obviously we can..but the question states the output should be the same as in the previous task which was an array of these product object. But still I will make the change you suggested.

Reply:
@Goutam192002 Can you resubmit the output screenshot for this endpoint?